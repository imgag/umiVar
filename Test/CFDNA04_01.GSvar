##DESCRIPTION=m_ALT=Multi-UMI alternative read counts
##DESCRIPTION=Strand=Alleles in strands: Alternative forward, Alternative reverse, Reference forward, Reference reverse
##DESCRIPTION=Sequence context=Sequence +/- 5bp around the variant.
##DESCRIPTION=Homopolymer=Indicates if variant is in a homopolymer region.
##DESCRIPTION=somatic_classification_comment=Somatic classification comment from the NGSD.
##DESCRIPTION=somatic_classification=Somatic classification from the NGSD.
##DESCRIPTION=normal_freq=Variant frequency.
##DESCRIPTION=normal_depth=Sequencing depth at the variant position.
##FILTER=Clustered_Variant=Clustered variants
##FILTER=Error=Alternative counts inside the expected error rate distribution
##FILTER=Fisher_Strand=Strand bias based on fisher test
##FILTER=LC_Downstream=Low complexity region (5bps) downstream. ≥ 80% of bases show the same nucleotide  or tandem of ≥ 3 equal nucleotides in a row
##FILTER=LC_Upstream=Low complexity region (5bps) upstream. ≥ 80% of bases show the same nucleotide or tandem of ≥ 3 equal nucleotides in a row
##FILTER=Low_AC=Less than defined minimum of alternative counts
##FILTER=Low_COV=Low coverage
##FILTER=Low_qual_pos=Position enriched with too many low quality bases
##FILTER=No_Coverage=No read coverage for this monitoring/ID variant in BAM file.
##FILTER=PASS=Passed filter
##FILTER=Strand_imbalanced=All alternative reads found in only one strand
##FILTER=Variant_Contamination=no description available
##FILTER=Variant_contamination=Reads supporting other alleles outside of the error rate distribution
##FILTER=low_conf_region=Low confidence region for small variant calling based on gnomAD AC0/RF filters and IMGAG trio/twin data.
##FILTER=off-target=Variant marked as 'off-target'.
#chr	start	end	ref	obs	tumor_af	tumor_dp	p-value	Alt counts	m_AF	m_REF	m_ALT	Strand	Sequence context	Homopolymer	filter	quality	gene	normal_freq	normal_depth
chr1	752894	752894	G	A	0.1344	8685	6.73E-14	1167	0.13026	5355	802	541-626-3587-3918	TTCTGGATCCA	false		QUAL=0	MPL	0	1493
chr1	762273	762273	G	A	0.1097	7586	1.04E-12	832	0.10651	4807	573	472-360-3081-3672	CCCTCGGAAGG	false		QUAL=0	MUTYH	0	1520
chr1	762589	762589	G	A	0.0512	9618	7.08E-10	492	0.05145	6453	350	236-256-5110-4005	GCCAGGGATCT	false		QUAL=0	PIK3C2B	0	2131
chr1	762592	762592	C	T	0	0		0	0	0	0	0-0-0-0	.	false	No_Coverage	QUAL=0	SOS1	0	893
chr1	762601	762601	G	A	0.1571	8708	7.71E-15	1368	0.15202	5489	984	577-791-3363-3968	AACTCGAAATA	false		QUAL=0	LRP1B	0	578
chr1	792263	792263	A	T	0.068	10516	0.00E+00	715	0.06609	7207	510	377-338-6269-3530	ATTCTAATTAT	false		QUAL=0	KIT	0	1307
chr1	792480	792480	G	A	0.1512	11836	4.65E-15	1789	0.15219	7192	1291	1037-752-6162-3874	CTGATGAAATG	false		QUAL=0	TERT	0.0005	2169
chr1	866319	866319	C	T	0.0627	7531	1.11E-10	472	0.05893	5414	339	221-251-3271-3770	AGTTTCCTCTT	false	Variant_Contamination	QUAL=0	TNPO1	0	1172
chr1	866511	866511	C	T	0.0554	10878	6.05E-10	603	0.05379	7652	435	332-271-5922-4348	GTCATCGTGAA	false		QUAL=0	RAD50	0	549
chr1	871334	871334	G	A	0	0		0	0	0	0	0-0-0-0	.	false	No_Coverage	QUAL=0	ITK	0	1336
chr1	876499	876499	C	T	0.1093	8401	8.47E-13	918	0.10585	5575	660	357-561-2785-4693	CATTTCCTGGA	false		QUAL=0	RAC1	0	1907
chr1	877715	877715	C	T	0.0622	9691	1.54E-10	603	0.05989	6609	421	238-365-4087-4995	TGTTACCTGAG	false		QUAL=0	KMT2C	0	1479
chr1	877831	877831	G	A	0.1264	9825	1.26E-13	1242	0.12879	5892	871	512-730-3684-4887	ACACTGATTAT	false		QUAL=0	KMT2C	0	1858
chr1	879676	879676	G	A	0.1304	9980	5.82E-14	1301	0.13347	6356	979	534-767-3381-5290	AGTTCGAACTT	false		QUAL=0	KAT6A	0	1556
chr1	879687	879687	C	T	0	0		0	0	0	0	0-0-0-0	.	false	No_Coverage	QUAL=0	PTPRD	0	1298
chr1	880238	880238	C	T	0	0		0	0	0	0	0-0-0-0	.	false	No_Coverage	QUAL=0	ANK3	0	1481
chr1	881627	881627	G	A	0	0		0	0	0	0	0-0-0-0	.	false	No_Coverage	QUAL=0	ANK3	0	2102
chr1	883625	883625	G	A	0	0		0	0	0	0	0-0-0-0	.	false	No_Coverage	QUAL=0	JMJD1C	0	884
chr1	883899	883899	C	T	0	0		0	0	0	0	0-0-0-0	.	false	No_Coverage	QUAL=0	WT1	0	1236
chr1	884091	884091	C	T	0	0		0	0	0	0	0-0-0-0	.	false	No_Coverage	QUAL=0	PTPRB	0	1786
chr1	884091	884091	G	A	0	0		0	0	0	0	0-0-0-0	.	false	No_Coverage	QUAL=0	NCOR2	0	620
chr1	887560	887560	C	T	0.1248	17070	1.22E-13	2131	0.12365	7810	1102	358-1773-2414-12519	GATTTCCCACA	false		QUAL=0	FLT3	0	1860
chr1	887801	887801	C	T	0	0		0	0	0	0	0-0-0-0	.	false	No_Coverage	QUAL=0	FLT1	0	1929
chr1	888639	888639	G	A	0.065	8743	1.07E-10	568	0.06538	5975	418	258-310-3409-4757	CAGTCGATGGG	false		QUAL=0	CHD8	0	1504
chr1	888659	888659	G	-	0	0		0	0	0	0	0-0-0-0	.	false	No_Coverage	QUAL=0	AXIN1	0	1034
chr1	889158	889158	C	T	0.0415	10089	2.16E-09	419	0.03888	7046	285	160-259-3159-6499	AGGATCCATAG	false		QUAL=0	NF1	0	1989
chr1	889159	889159	T	G	0.059	8931	3.40E-21	527	0.06151	5996	393	246-281-3198-5204	AATTGTGCAGA	false		QUAL=0	NF1	0	1157
chr1	892745	892745	G	A	0	0		0	0	0	0	0-0-0-0	.	false	No_Coverage	QUAL=0	STAT3	0	1923
chr1	894573	894573	C	T	0.105	9724	1.55E-12	1021	0.1038	6329	733	341-680-3348-5349	TTCTTCGTGAA	false		QUAL=0	CLTC	0.0006	1640
chr1	897325	897325	C	T	0.0418	9781	1.04E-09	409	0.04065	6797	288	176-233-4721-4647	AATACCAGGAC	false		QUAL=0	ARHGAP35	0	1801
