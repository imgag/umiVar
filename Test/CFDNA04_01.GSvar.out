##DESCRIPTION=m_ALT=Multi-UMI alternative read counts
##DESCRIPTION=Strand=Alleles in strands: Alternative forward, Alternative reverse, Reference forward, Reference reverse
##DESCRIPTION=Sequence context=Sequence +/- 5bp around the variant.
##DESCRIPTION=Homopolymer=Indicates if variant is in a homopolymer region.
##DESCRIPTION=somatic_classification_comment=Somatic classification comment from the NGSD.
##DESCRIPTION=somatic_classification=Somatic classification from the NGSD.
##DESCRIPTION=normal_freq=Variant frequency.
##DESCRIPTION=normal_depth=Sequencing depth at the variant position.
##FILTER=Clustered_Variant=Clustered variants
##FILTER=Error=Alternative counts inside the expected error rate distribution
##FILTER=Fisher_Strand=Strand bias based on fisher test
##FILTER=LC_Downstream=Low complexity region (5bps) downstream. ≥ 80% of bases show the same nucleotide  or tandem of ≥ 3 equal nucleotides in a row
##FILTER=LC_Upstream=Low complexity region (5bps) upstream. ≥ 80% of bases show the same nucleotide or tandem of ≥ 3 equal nucleotides in a row
##FILTER=Low_AC=Less than defined minimum of alternative counts
##FILTER=Low_COV=Low coverage
##FILTER=Low_qual_pos=Position enriched with too many low quality bases
##FILTER=No_Coverage=No read coverage for this monitoring/ID variant in BAM file.
##FILTER=PASS=Passed filter
##FILTER=Strand_imbalanced=All alternative reads found in only one strand
##FILTER=Variant_Contamination=no description available
##FILTER=Variant_contamination=Reads supporting other alleles outside of the error rate distribution
##FILTER=low_conf_region=Low confidence region for small variant calling based on gnomAD AC0/RF filters and IMGAG trio/twin data.
##FILTER=off-target=Variant marked as 'off-target'.
#chr	start	end	ref	obs	tumor_af	tumor_dp	p-value	Alt counts	m_AF	m_REF	m_ALT	Strand	Sequence context	Homopolymer	filter	quality	gene	normal_freq	normal_depth	Tumor_Filter	Post_Filter
chr1	752894	752894	G	A	0.1344	8685	6.73e-14	1167	0.13026	5355	802	541-626-3587-3918	TTCTGGATCCA	False		QUAL=0	MPL	0.0	1493	freq-tum;HIGH_POP_AF	TUMOR_FILTER;
chr1	762273	762273	G	A	0.1097	7586	1.04e-12	832	0.10651	4807	573	472-360-3081-3672	CCCTCGGAAGG	False		QUAL=0	MUTYH	0.0	1520	PASS;HIGH_POP_AF	TUMOR_FILTER;
chr1	762589	762589	G	A	0.0512	9618	7.08e-10	492	0.05145	6453	350	236-256-5110-4005	GCCAGGGATCT	False		QUAL=0	PIK3C2B	0.0	2131	PASS;HIGH_POP_AF	TUMOR_FILTER;
chr1	762592	762592	C	T	0.0	0		0	0.0	0	0	0-0-0-0	.	False	No_Coverage	QUAL=0	SOS1	0.0	893	freq-tum;HIGH_POP_AF	LOW_mALT_COUNT;LOW_DEPTH;STRAND_BIAS;TUMOR_FILTER;NON-INFORMATIVE;LOW_MULTIDEPTH;
chr1	762601	762601	G	A	0.1571	8708	7.71e-15	1368	0.15202000000000002	5489	984	577-791-3363-3968	AACTCGAAATA	False		QUAL=0	LRP1B	0.0	578		
chr1	792263	792263	A	T	0.068	10516	0.0	715	0.06609	7207	510	377-338-6269-3530	ATTCTAATTAT	False		QUAL=0	KIT	0.0	1307		
chr1	792480	792480	G	A	0.1512	11836	4.65e-15	1789	0.15219000000000002	7192	1291	1037-752-6162-3874	CTGATGAAATG	False		QUAL=0	TERT	0.0005	2169		
chr1	866319	866319	C	T	0.0627	7531	1.11e-10	472	0.058929999999999996	5414	339	221-251-3271-3770	AGTTTCCTCTT	False	Variant_Contamination	QUAL=0	TNPO1	0.0	1172	off-target	TUMOR_FILTER;
chr1	866511	866511	C	T	0.0554	10878	6.05e-10	603	0.05379	7652	435	332-271-5922-4348	GTCATCGTGAA	False		QUAL=0	RAD50	0.0	549	freq-tum;off-target	TUMOR_FILTER;
chr1	871334	871334	G	A	0.0	0		0	0.0	0	0	0-0-0-0	.	False	No_Coverage	QUAL=0	ITK	0.0	1336		LOW_mALT_COUNT;LOW_DEPTH;STRAND_BIAS;NON-INFORMATIVE;LOW_MULTIDEPTH;
chr1	876499	876499	C	T	0.1093	8401	8.47e-13	918	0.10585	5575	660	357-561-2785-4693	CATTTCCTGGA	False		QUAL=0	RAC1	0.0	1907		
chr1	877715	877715	C	T	0.0622	9691	1.54e-10	603	0.05989	6609	421	238-365-4087-4995	TGTTACCTGAG	False		QUAL=0	KMT2C	0.0	1479		
chr1	877831	877831	G	A	0.1264	9825	1.26e-13	1242	0.12879000000000002	5892	871	512-730-3684-4887	ACACTGATTAT	False		QUAL=0	KMT2C	0.0	1858	freq-tum;off-target;HIGH_POP_AF	TUMOR_FILTER;
chr1	879676	879676	G	A	0.1304	9980	5.82e-14	1301	0.13347	6356	979	534-767-3381-5290	AGTTCGAACTT	False		QUAL=0	KAT6A	0.0	1556	freq-tum	TUMOR_FILTER;
chr1	879687	879687	C	T	0.0	0		0	0.0	0	0	0-0-0-0	.	False	No_Coverage	QUAL=0	PTPRD	0.0	1298		LOW_mALT_COUNT;LOW_DEPTH;STRAND_BIAS;NON-INFORMATIVE;LOW_MULTIDEPTH;
chr1	880238	880238	C	T	0.0	0		0	0.0	0	0	0-0-0-0	.	False	No_Coverage	QUAL=0	ANK3	0.0	1481	PASS;HIGH_POP_AF	LOW_mALT_COUNT;LOW_DEPTH;STRAND_BIAS;TUMOR_FILTER;NON-INFORMATIVE;LOW_MULTIDEPTH;
chr1	881627	881627	G	A	0.0	0		0	0.0	0	0	0-0-0-0	.	False	No_Coverage	QUAL=0	ANK3	0.0	2102	freq-tum	LOW_mALT_COUNT;LOW_DEPTH;STRAND_BIAS;TUMOR_FILTER;NON-INFORMATIVE;LOW_MULTIDEPTH;
chr1	883625	883625	G	A	0.0	0		0	0.0	0	0	0-0-0-0	.	False	No_Coverage	QUAL=0	JMJD1C	0.0	884	LowEVS;off-target	LOW_mALT_COUNT;LOW_DEPTH;STRAND_BIAS;TUMOR_FILTER;NON-INFORMATIVE;LOW_MULTIDEPTH;
chr1	883899	883899	C	T	0.0	0		0	0.0	0	0	0-0-0-0	.	False	No_Coverage	QUAL=0	WT1	0.0	1236	freq-tum;off-target	LOW_mALT_COUNT;LOW_DEPTH;STRAND_BIAS;TUMOR_FILTER;NON-INFORMATIVE;LOW_MULTIDEPTH;
chr1	884091	884091	C	T	0.0	0		0	0.0	0	0	0-0-0-0	.	False	No_Coverage	QUAL=0	PTPRB	0.0	1786		LOW_mALT_COUNT;LOW_DEPTH;STRAND_BIAS;NON-INFORMATIVE;LOW_MULTIDEPTH;
chr1	884091	884091	G	A	0.0	0		0	0.0	0	0	0-0-0-0	.	False	No_Coverage	QUAL=0	NCOR2	0.0	620		LOW_mALT_COUNT;LOW_DEPTH;STRAND_BIAS;NON-INFORMATIVE;LOW_MULTIDEPTH;
chr1	887560	887560	C	T	0.1248	17070	1.22e-13	2131	0.12365	7810	1102	358-1773-2414-12519	GATTTCCCACA	False		QUAL=0	FLT3	0.0	1860		
chr1	887801	887801	C	T	0.0	0		0	0.0	0	0	0-0-0-0	.	False	No_Coverage	QUAL=0	FLT1	0.0	1929		LOW_mALT_COUNT;LOW_DEPTH;STRAND_BIAS;NON-INFORMATIVE;LOW_MULTIDEPTH;
chr1	888639	888639	G	A	0.065	8743	1.0699999999999999e-10	568	0.06538	5975	418	258-310-3409-4757	CAGTCGATGGG	False		QUAL=0	CHD8	0.0	1504		
chr1	888659	888659	G	-	0.0	0		0	0.0	0	0	0-0-0-0	.	False	No_Coverage	QUAL=0	AXIN1	0.0	1034		LOW_mALT_COUNT;INDEL;LOW_DEPTH;STRAND_BIAS;NON-INFORMATIVE;LOW_MULTIDEPTH;
chr1	889158	889158	C	T	0.0415	10089	2.1600000000000004e-09	419	0.03888	7046	285	160-259-3159-6499	AGGATCCATAG	False		QUAL=0	NF1	0.0	1989	LowEVS;off-target	TUMOR_FILTER;
chr1	889159	889159	T	G	0.059000000000000004	8931	3.4e-21	527	0.06151	5996	393	246-281-3198-5204	AATTGTGCAGA	False		QUAL=0	NF1	0.0	1157	freq-tum	TUMOR_FILTER;
chr1	892745	892745	G	A	0.0	0		0	0.0	0	0	0-0-0-0	.	False	No_Coverage	QUAL=0	STAT3	0.0	1923	off-target	LOW_mALT_COUNT;LOW_DEPTH;STRAND_BIAS;TUMOR_FILTER;NON-INFORMATIVE;LOW_MULTIDEPTH;
chr1	894573	894573	C	T	0.105	9724	1.55e-12	1021	0.1038	6329	733	341-680-3348-5349	TTCTTCGTGAA	False		QUAL=0	CLTC	0.0006	1640	freq-tum;off-target	TUMOR_FILTER;
chr1	897325	897325	C	T	0.0418	9781	1.0400000000000001e-09	409	0.04065	6797	288	176-233-4721-4647	AATACCAGGAC	False		QUAL=0	ARHGAP35	0.0	1801	freq-tum;off-target	TUMOR_FILTER;
